src/libs/fileDownload/getImageResolution.native.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns image dimensions for image selected from react-native-image-picker
```

Comment:
Please add more of an explanation here of why it's OK for this method to be a no-op (since the height and width are already included with the image object)

Reply:
updated